### PR TITLE
Add UIView extensions for constrainting views via NSLayoutAnchors

### LINKED ISSUE #34: Add UIView extensions for constrainting views via NSLayoutAnchors
Layout Anchors is a mechanism introduced by Apple in iOS 9. It is a whole new way of adding constraints that is readable and concise.

NSLayoutAnchor class provides several advantages over using the NSLayoutConstraint API directly.
- The code is cleaner, more concise, and easier to read.
- The NSLayoutConstraint.Attribute subclasses provide additional type checking, preventing you from creating invalid constraints.

### PR DESCRIPTION
**Pull request by 1nk0gn1to**
Closes #34

### CODE REVIEW

**Comment by evgenyvalavin** [comment]
#34 says "[Add UIView extensions for constrainting views via NSLayoutAnchors](https://github.com/ArbusBiz/Arbus.Mobile.Essentials/issues/34)"

**Comment by evgenyvalavin** [comment]
We better create separate PRs for non NSLayoutAnchor extensions

**Review by evgenyvalavin — CHANGES_REQUESTED** [review]
We better create separate PRs for non NSLayoutAnchor extensions

**Comment by evgenyvalavin** [comment]
What about the PR name?

File: src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs [hunk 1]
```diff
@@ -223,4 +223,126 @@ private const int
...
+    }
+    
+    public static TView ConstraintRightTo<TView>(this TView view, NSLayoutAnchor<NSLayoutXAxisAnchor> anchor, nfloat constant = default)
+        where TView : UIView
+    {
+        view.RightAnchor.ConstraintEqualTo(anchor, -constant).Active = true;
+        return view;
+    }
+
+    public static NSLayoutConstraint Activate(this NSLayoutConstraint constraint)
```
Comment (evgenyvalavin) [inline src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs]: These are NSLayoutConstraintExtensions. We should put them in a NSLayoutConstraintExtensions.cs class

**Comment by 1nk0gn1to** [comment]
> What about the PR name?

I fixed it

File: src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs [hunk 2]
```diff
@@ -223,4 +223,126 @@ private const int
...
+
+    public static TView ConstraintHeightTo<TView>(this TView view, nfloat constant) where TView : UIView
+    {
+        view.HeightAnchor.ConstraintEqualTo(constant).Active = true;
+        return view;
+    }
+    
+    public static TView ConstraintHeightTo<TView>(this TView view, NSLayoutDimension anchor, nfloat multiplier = default, nfloat constant = default) where TView : UIView
+    {
+        if (multiplier == default)
```
Comment (evgenyvalavin) [inline src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs]: What if we set "1" as the default value?

**Review by evgenyvalavin — CHANGES_REQUESTED** [review]

File: src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs (hunk 2 above)
Comment (1nk0gn1to) [reply src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs]: I couldn't do it, because appearing error "need using constant value"

Comment (evgenyvalavin) [reply src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs]: Hm, let's use float for multiplier then

File: src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs (hunk 1 above)
Comment (1nk0gn1to) [reply src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs]: Ok

File: src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs (hunk 2 above)
Comment (1nk0gn1to) [reply src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs]: Done!

File: src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs (hunk 1 above)
Comment (1nk0gn1to) [reply src/iOS.Essentials/BasicViewExtensions/UIViewExtensions.cs]: Done!

**Comment by evgenyvalavin** [comment]
Good job!

**Review by evgenyvalavin — APPROVED** [review]

**Pull request merged**